SpatialGDK/Source/SpatialGDK/Private/Utils/ComponentFactory.cpp

```diff
@@ -314,9 +316,13 @@ void ComponentFactory::AddProperty(Schema_Object* Object, Schema_FieldId FieldId
 	{
 		// These properties can be set to replicate, but won't serialize across the network.
 	}
+	else if (Property->IsA<UMapProperty>())
+	{
+		UE_LOG(LogComponentFactory, Error, TEXT("Replicated TMaps are not supported."));
```

Comment:
Would be good to indicate to the user which property this is (variable name, class etc.)